doc/05_Plugins/README.md

```diff
@@ -88,10 +103,12 @@ Encore
 
   // Important! Reference this vendor-manifest in your build.
   // It will take care of injecting Ant-Design, React, etc. without the need to bundle it in your plugin.
-  .addPlugin(new webpack.DllReferencePlugin({
+  // Note: This is currently not even possible, since the build dir changes with every build (uidv4) and even if you
+  // use the right folder, it fails... is it really needed?
+  /*.addPlugin(new webpack.DllReferencePlugin({
     context: __dirname,
     manifest: path.join(__dirname, 'node_modules', 'pimcore-studio-ui', 'dist', 'vendor',  'vendor-manifest.json')
-  }))
+  }))*/
```

Comment:
In general it's needed to ensure that always the same copy of react is used through the whole app. 
Otherwise react will start to complain latest at the point where you are using some of our react hooks in your plugin. Also it reduces the bundled size of the plugins since webpack is using a dynamic link to a shared module instead of bundling a new copy into every plugin.

The uuid folders shouldn't affect you. Have you referenced this file from the installed npm package of studio in your plugin? Can you share some more information what is exactly failing in this case?

In general webpack is mapping the imported modules over deterministic ids at runtime. So, even when the output files are placed somewhere completely different - webpack will be able to map modules due to these ids.

Reply:
thanks for the response.

I figured it out now. I was confused due to the npm install of the studio ui. I though I needed to reference the actual build files in the public folder of the Studio UI Bundle, but I need to reference the node_modules studio ui bundle. with that, this is working now.